--- sweep-0.9.2/sweep.desktop~	2008-03-26 23:01:35.000000000 +0100
+++ sweep-0.9.2/sweep.desktop	2008-03-26 23:05:39.000000000 +0100
@@ -1,11 +1,14 @@
 [Desktop Entry]
 Name=Sweep
+Name[pl]=Sweep
 GenericName=Sound Editor
 Comment=Sweep Sound Editor
-Categories=Application;AudioVideo;
+Comment[pl]=Edytor dźwięku
+Categories=GTK;Audio;Recorder;
 Encoding=UTF-8
 Exec=sweep
 Icon=sweep.svg
 MimeType=audio/x-wav
 Terminal=false
 Type=Application
+# vi: encoding=utf-8
